easybuild/easyconfigs/s/segemehl/segemehl-0.3.4-iccifort-2020.4.304.eb

```
@@ -0,0 +1,36 @@
+easyblock = 'MakeCp'
+
+name = "segemehl"
+version = "0.3.4"
+
+homepage = 'https://www.bioinf.uni-leipzig.de/Software/segemehl/'
+description = """segemehl is a software to map short sequencer reads to reference genomes. 
+ Unlike other methods, segemehl is able to detect not only mismatches but also insertions 
+ and deletions. Furthermore, segemehl is not limited to a specific read length and is able 
+ to mapprimer- or polyadenylation contaminated reads correctly. segemehl implements a matching 
```

```suggestion
 to map primer- or polyadenylation contaminated reads correctly. segemehl implements a matching 
```
Typo in the original, fixed at e.g. https://nodepit.com/node/de.helmholtz_muenchen.ibis.ngs.segemehl.SegemehlNodeFactory